src/languages/javascript-repl.js

```diff
@@ -0,0 +1,30 @@
+/*
+Language: JavaScript REPL
+Requires: javascript.js
+Author: Marat Nagayev <nagaevmt@yandex.ru>
+Category: common
+*/
+
+export default function(hljs) {
+    return {
+      name: 'JavaScript REPL',
+      contains: [
```

Comment:
```suggestion
      name: 'JavaScript REPL',
      aliases: ["js-repl"],
      contains: [
```

IF we keep the name I think this would be desirable.